tn_api_hotel.yml

```diff
@@ -863,3 +938,136 @@ components:
                                         description: The url for the image of the above titled part of the hotel of size thumbnail.
                                         example: https://media-cdn.tripadvisor.com/media/photo-m/1280/1b/45/0b/f8/lobby.jpg
 
+
+
+    HotelRulesRequest:
+      required:
+        - hotel_itinerary_reference
+        - hotel_trace_id
+        - hotel_chain
+        - hotel_code
+        - rate_plan_type
+        - base_price_incl_curr
+        - traveller_total
+        - rooms_total
+        - check_in_date
+        - check_out_date
+        - credential_info
+      properties:
+        hotel_itinerary_reference:
+          type: string
+          example: '1'
+          description: A reference that indicates the itineraries position in the flow of itineraries in the super_trip with the index starting at 0.
+        hotel_trace_id:
+          type: string
+          example: 'ndsf6sjf6sdf7sdf6'
+          description: An identifier to refer to the hotel for searching room availability
+        hotel_chain:
+          type: string
+          description: The refence for the hotel chain.
+          example: 'GI'
+        hotel_code:
+          type: string
+          description: The reference for the hotel code
+          example: '35573'
+        rate_plan_type:
+          type: string
+          description: The reference for the room and rate plan selection
+          example: 'A01LV8'
+        base_price_incl_curr:
+          type: string
+          description: The total price of the room including the currency, as returned from hotel details
+          example: 'USD1234.56'
+        traveller_total:
+          type: int
+          example: 3
+          description: Number of people that will be staying at the hotel.
+        rooms_total:
+          type: int
+          example: 2
+          description: The number of rooms required for the above travellers.
+        check_in_date:
+          type: string
+          format: date
+          description: "Date of check-in. format: YYYY-MM-DD."
+        check_out_date:
+          type: string
+          format: date
+          description: "Date to checkout from the hotel. format: YYYY-MM-DD."
+        credential_info:
+          type: object
+          required:
+            - pcc
+            - provider
+            - pcc_currency
```

Comment:
The pcc currency isn't needed here or in the description of the credentials

Reply:
resolved